bedrock/base/templates/includes/global-nav.html

```diff
@@ -75,19 +70,17 @@ <h3 class="summary" data-id="firefox">
               <ul class="nav-menu-secondary-links">
                 <li>
                 {% if LANG == 'en-US' %}
```

Comment:
I debated this when I reviewed the PR that introduced it. My reasoning was that since the strings were already wrapped, and we (at the time, anyway) seemed to be going in the direction of showing the global nav to all, I figured it was appropriate future-proofing.

I think leaving it in place for now makes sense.